pkg/handler/http/contact.go

```diff
@@ -0,0 +1,44 @@
+package http
+
+import (
+	"context"
+	"github.com/CollActionteam/collaction_backend/internal/contact"
+	"github.com/CollActionteam/collaction_backend/internal/models"
+	"github.com/CollActionteam/collaction_backend/pkg/repository"
+	"github.com/gin-gonic/gin"
+	"net/http"
+)
+
+type ContactService interface {
+	SendEmail(ctx context.Context, data models.EmailContactRequest) error
+}
+
+type ContactHandler struct {
+	service ContactService
+}
+
+func NewContactHandler() *ContactHandler {
+	emailRepository := repository.NewEmail("userName", "password")
+	configManager := repository.NewConfigManager()
```

Comment:
why config manager is in the repository layer?

Reply:
to use config we need to call an external service (SSM), so the repository is the place to handle externals,